sample-package-core/src/main/resources/OSGI-INF/custombulkaction-bulk-action-contrib.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<component name="org.nuxeo.sample.core.actions.custombulkaction">
+
+  <require>org.nuxeo.ecm.core.bulk</require>
+
+  <extension target="org.nuxeo.ecm.core.bulk" point="actions">
+    <action name="custombulkaction" bucketSize="100" batchSize="25" sequentialCommands="true"/>
```

Comment:
```    <action name="myCustomAction" inputStream="bulk/myCustomAction" bucketSize="100" batchSize="25" />
```
again adding some comment could help:
```
<!-- bucketSize is the number of document ids that will be put in the action record, batchSize is the number of document ids that is going to be processed within a transaction, we always want to have bucketSize >= batchSize -->
```

Reply:
Understood, will add the comment.
What inputStream="bulk/myCustomAction" is doing though? I'm not familiar with it